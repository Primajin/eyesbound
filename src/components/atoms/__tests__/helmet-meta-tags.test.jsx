import React from 'react';
import {create} from 'react-test-renderer';

import HelmetMetaTags from '../helmet-meta-tags.jsx';

describe('HelmetMetaTags', () => {
	it('renders correctly without props', () => {
		const component = create(<HelmetMetaTags/>);
		expect(component).toMatchSnapshot();
	});

	it('renders correctly with props', () => {
		const coordinates = [52.473_092, 13.327_628];
		const dateTime = '1986-03-14T11:25:00+0000';
		const description = 'Anyone can hold the helm when the sea is calm';
		const imageSource = 'https://images.prismic.io/eyesbound/d4b035f1-21c8-476a-bd8f-6335f44ced7d__DSC2116.jpg';
		const path = 'picture';
		const title = 'Libeskind';
		const uid = 'libeskind';
		const component = create(<HelmetMetaTags
			coordinates={coordinates}
			dateTime={dateTime}
			description={description}
			imageSource={imageSource}
			path={path}
			title={title}
			uid={uid}
		/>);
		expect(component).toMatchSnapshot();
	});
});
