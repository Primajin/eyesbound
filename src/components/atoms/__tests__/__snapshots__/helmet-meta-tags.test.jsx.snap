// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`HelmetMetaTags renders correctly with props 1`] = `
<div
  className="mock-helmet"
>
  <title>
    Libeskind | EYESBOUND
  </title>
  <meta
    content="Libeskind | EYESBOUND"
    name="title"
  />
  <meta
    content="Libeskind | EYESBOUND"
    property="og:title"
  />
  <meta
    content="Libeskind | EYESBOUND"
    name="twitter:title"
  />
  <meta
    content="Anyone can hold the helm when the sea is calm"
    property="og:description"
  />
  <meta
    content="Anyone can hold the helm when the sea is calm"
    name="twitter:description"
  />
  <meta
    content="https://images.prismic.io/eyesbound/d4b035f1-21c8-476a-bd8f-6335f44ced7d__DSC2116.jpg"
    property="og:image"
  />
  <meta
    content="https://images.prismic.io/eyesbound/d4b035f1-21c8-476a-bd8f-6335f44ced7d__DSC2116.jpg"
    name="twitter:image"
  />
  <link
    href="https://eyesbound.com/picture/libeskind"
    rel="canonical"
  />
  <meta
    content="https://eyesbound.com/picture/libeskind"
    property="og:url"
  />
  <meta
    content="52.473092;13.327628"
    name="geo.position"
  />
  <meta
    content="52.473092, 13.327628"
    name="ICBM"
  />
  <meta
    content="1986-03-14T11:25:00+0000"
    name="date"
  />
</div>
`;

exports[`HelmetMetaTags renders correctly without props 1`] = `
<div
  className="mock-helmet"
/>
`;
